dask_ml/model_selection/_incremental.py

```diff
@@ -692,6 +709,72 @@ class IncrementalSearch(BaseIncrementalSearch):
 
         If None, the estimator's default scorer (if available) is used.
 
+    Attributes
+    ----------
+    cv_results_ : dict of np.ndarrays
+        This dictionary has keys
+
+        * ``mean_partial_fit_time``
+        * ``mean_score_time``
+        * ``std_partial_fit_time``
+        * ``std_score_time``
+        * ``test_score``
+        * ``rank_test_score``
+        * ``model_id``
+        * ``partial_fit_calls``
+        * ``params``
+        * ``param_{key}`` for every ``key`` in the parameters for that model.
```

Comment:
Rather than "parameters for that model", should it be "hyperparameters in `params`"?

Reply:
Or "where `key` is every key in `params`"?

Reply:
Scikit-learn gives an example `cv_results_` in their docs, and don't address this generally.